pkg/auth/rbac.go

```diff
@@ -173,12 +173,12 @@ func ValidateRbacPermission(line string) (p *Permission, err error) {
 	}, nil
 }
 
-func ReadRbacPolicy(dexEnabled bool) (policy map[string]*Permission, err error) {
+func ReadRbacPolicy(dexEnabled bool, dexRbacPolicy string) (policy map[string]*Permission, err error) {
```

Comment:
```suggestion
func ReadRbacPolicy(dexEnabled bool, dexRbacPolicyPath string) (policy map[string]*Permission, err error) {
```

Reply:
Changed the naming. Will resolve this issue.